client/lib/plugins/actions.js

```diff
@@ -256,7 +256,8 @@ const PluginsActions = {
 
 		const manageError = error => {
 			if ( error.name === 'PluginAlreadyInstalledError' ) {
-				if ( site.isMainNetworkSite() ) {
+				//compatibility with old site object (for now)
```

Comment:
Maybe leave a few TODO's with the same message so we can easily search for references we need to update.